source/team.html.erb

```diff
@@ -99,7 +99,7 @@ responsive: true
     The Ember Data Team
   </h2>
   <p class="lead">
-    
+  The Ember Data Team is responsible for (responsibilities). The team meets on (day), and the public notes can be found <a href="github.com/ember-data">on Github</a>.
```

Comment:
Ember Data is the official data persistence library for Ember.js applications.

Reply:
The team meets on Wednesdays at (12pm EST), and the public notes can be found at [https://github.com/emberjs/core-notes/tree/master/ember-data](https://github.com/emberjs/core-notes/tree/master/ember-data)